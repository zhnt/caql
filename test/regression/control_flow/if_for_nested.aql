// If statements nested inside for loops
let sum = 0

for i = 1, 10, 1 {
    if i % 2 == 0 {
        print("Even: " + string(i))
        sum = sum + i
    } else {
        print("Odd: " + string(i))
    }
}

print("Sum of evens: " + string(sum))
