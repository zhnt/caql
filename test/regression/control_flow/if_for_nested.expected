Odd: 1
Even: 2
Odd: 3
Even: 4
Odd: 5
Even: 6
Odd: 7
Even: 8
Odd: 9
Sum of evens: 20
